ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss

```diff
@@ -13,10 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@mixin subtitle-typography() {
+@mixin body-strong-typography() {
```

Comment:
normally this still exist : 
![image](https://user-images.githubusercontent.com/4974420/153830713-96b3cb1c-1e2f-4f0f-8ec6-c218c9793aef.png)